DFC.ServiceTaxonomy.GraphSync/Handlers/Orchestrators/DeleteOrchestrator.cs

```diff
@@ -45,10 +54,19 @@ public async Task<bool> Unpublish(ContentItem contentItem)
             // no need to touch the draft graph, there should always be a valid version in there
             // (either a separate draft version, or the published version)
 
-            return await DeleteFromGraphReplicaSetIfAllowed(
+            if (!await DeleteFromGraphReplicaSetIfAllowed(
                 contentItem,
                 _publishedContentItemVersion,
-                DeleteOperation.Unpublish);
+                DeleteOperation.Unpublish))
+            {
+                return false;
+            }
+
+            //if unpublish was successful publish events
+            await PublishContentEvent(contentItem, ContentEventType.Unpublished);
```

Comment:
The orchestrators need to call the IContentOrchestrationHandlers, instead of publishing the events directly.

So in effect, these code snippets from `GraphSyncContentHandler`, need to be removed from there and moved down in to the orchestrator....

```
            foreach (var contentOrchestrationHandler in _contentOrchestrationHandlers)
            {
                await contentOrchestrationHandler.DraftSaved(context.ContentItem);
            }
```

Reply:
Done!